### PR TITLE
run 0.6.5 (new formula)

### PR DESCRIPTION
**Pull request by dawidd6**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

> Run aims to be better at managing small scripts and wrappers, while incorporating a familiar make-like syntax.

https://github.com/TekWizely/run/issues/6

### CODE REVIEW

**Comment by dawidd6** [comment]
There is one problem during installation though:
```
Warning: Generic binaries were installed to "/home/linuxbrew/.linuxbrew/opt/run/bin"
Binaries with generic names are likely to conflict with other software,
and suggest that this software should be installed to "libexec" and then
symlinked as needed.
```

**Comment by TekWizely** [comment]
Hi @dawidd6,

First, thanks so much for helping get this started !

I have create a few releases since our first discussion, with current being `v0.6.2` which contains a few useful enhancements.

Q: Could we update this PR to that version, or would it be better to wait until this initial version is in and make an update from there?

Lemme know what you think and also if there are any next steps for me to keep this morning.

Thanks again !

-TW

**Comment by dawidd6** [comment]
Sure, I'll update this PR once I have time and if it won't get merged in the meantime.

**Comment by fxcoudert** [comment]
Ready to merge, except the commit message should have the right version in it :)

File: Formula/run.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Run < Formula
+  desc "Easily manage and invoke small scripts and wrappers"
+  homepage "https://github.com/TekWizely"
```
Comment (equal-l2) [inline Formula/run.rb]:
```suggestion
  homepage "https://github.com/TekWizely/run"
```

`homepage` should point to the homepage for the product, not for the author.

Comment (dawidd6) [reply Formula/run.rb]: Thanks man, missed that.

**Pull request closed without merging**